src/ruby/lib/grpc/generic/rpc_server.rb

```diff
@@ -407,7 +413,13 @@ def loop_handle_server_calls
       request_call_tag = Object.new
       until stopped?
         deadline = from_relative_time(@poll_period)
-        an_rpc = @server.request_call(@cq, request_call_tag, deadline)
+        begin
+          an_rpc = @server.request_call(@cq, request_call_tag, deadline)
+        rescue Core::CallError, RuntimeError => e
+          # can happen during server shutdown
```

Comment:
This doesn't seem correct to me. I'm pretty sure that when a server shuts down, any outstanding `request_call` calls complete with an event with `success==true` and `call=null`. That doesn't look to me like one of the cases in `grpc_rb_server_request_call` that results in throwing an error.